tests/BenchmarkDotNet.Tests/ConfigParserTests.cs

```diff
@@ -383,6 +383,18 @@ public void NetMonikersAreRecognizedAsNetCoreMonikers(string tfm)
             Assert.Equal(tfm, ((DotNetCliGenerator)toolchain.Generator).TargetFrameworkMoniker);
         }
 
+        [Theory]
+        [InlineData("mono70", "net7.0")]
+        public void MonoMonikersAreRecognizedAsNetCoreMonikers(string tfm, string actualRuntimeId)
+        {
+            var config = ConfigParser.Parse(new[] { "-r", tfm }, new OutputLogger(Output)).config;
+
+            Assert.Single(config.GetJobs());
+            CsProjCoreToolchain toolchain = config.GetJobs().Single().GetToolchain() as CsProjCoreToolchain;
+            Assert.NotNull(toolchain);
+            Assert.Equal(actualRuntimeId, ((DotNetCliGenerator)toolchain.Generator).TargetFrameworkMoniker);
+        }
+
```

Comment:
Please add tests similar to [NativeAotTests](https://github.com/dotnet/BenchmarkDotNet/blob/master/tests/BenchmarkDotNet.IntegrationTests/NativeAotTests.cs) where you ask to run a `Mono .NET 6` benchmark (our CI uses .NET 6 SDK) and the benchmark just throws when it does not run on Mono. Like here where we do that for NativeAOT:

https://github.com/dotnet/BenchmarkDotNet/blob/1f5637c784ad9887a2d5abdba54d1337655281b1/tests/BenchmarkDotNet.IntegrationTests/NativeAotTests.cs#L55-L57